### DISCUSSION TITLE
The ouath page

### DISCUSSION THREAD

**Discussion by Dev4098** [community/community#120696] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The oauth page isn't responding I'm trying to connect my herkou with github but it's not helping whenever I'm going to oauth page it's only showing a blank site

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Dev4098 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
